frontend/src/router.jsx

```diff
@@ -13,7 +14,8 @@ class Router extends React.Component {
         <Switch>
           <Route exact path="/" component={HomePage} />
           <Route path="/restaurants-list" component={RestautantListPage} />
```

Comment:
change path according to route that you use for restaurant:id

Reply:
<Route path="/restaurants" exact component={RestautantListPage} />
          <Route path="/restaurants/:id" exact component={RestaurantPage} />
Something like this?